### PR TITLE
Fix creating free-form rubric comments

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-14039
affects: teacher
release note: Fixed creating free-form rubric comments

Test plan:
* Add a rubric comment using repro steps in [ticket](https://instructure.atlassian.net/browse/MBL-14039)

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/7unmtmg) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F7unmtmg)
teacher: [Link](https://rebrand.ly/cxxg727) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fcxxg727)
parent: [Link](https://rebrand.ly/nnzawit) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fnnzawit)

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024818&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed creating free-form rubric comments

#### Affected Apps: Teacher

[MBL-14039](https://instructure.atlassian.net/browse/MBL-14039)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.56% | 93.56% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/src/modules/speedgrader/GradeTab.js [hunk 1]
```diff
@@ -97,10 +97,10 @@ export class GradeTab extends Component<GradeTabProps, GradeTabState> {
     this.setState({ criterionCommentInput: criterionID })
   }
 
-  submitRubricComment = ({ message }: { message: string }) => {
+  submitRubricComment = ({ comment }) => {
```
Comment (vanwagonet) [inline rn/Teacher/src/modules/speedgrader/GradeTab.js]: yay for flow's not quite real type system not catching this oops years ago.

**Pull request merged**